src/Agent.Worker/CodeCoverage/CodeCoverageCommands.cs

```diff
@@ -282,7 +282,14 @@ private void ModifyCoberturaIndexDotHTML(string reportDirectory, IExecutionConte
                     File.Exists(nonFrameHtml))
                 {
                     // duplicating frame-summary.html to index.html and renaming index.html to newindex.html
-                    File.Delete(newIndexHtml);
+                    try
+                    {
+                        File.Delete(newIndexHtml);
+                    }
+                    catch
+                    {
+                        executionContext.Warning("Unable to delete old tracking folder");
```

Comment:
I'm not sure if users should know that. Can it be just a worker trace?
If it's needed for users, maybe describe here the impact for them in massage + add localization